android/app/src/main/java/com/shoutemapp/MainApplication.java

```diff
@@ -40,6 +44,10 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
           new MainReactPackage(),
+            new MapsPackage(),
+            new ImagePickerPackage(),
+            new PhotoViewPackage(),
+            new LinearGradientPackage(),
```

Comment:
This probably shouldn't be committed. Aren't those dependencies added by `react-native link`?